beacon-chain/utils/shuffle.go

```diff
@@ -70,3 +79,74 @@ func SplitIndices(l []uint64, n uint64) [][]uint64 {
 	}
 	return divided
 }
+
+// PermutedIndex Return `p(index)` in a pseudorandom permutation `p` of `0...list_size - 1` with ``seed`` as entropy.
+//     Utilizes 'swap or not' shuffling
```

Comment:
can you format the comments?

Reply:
+1

Reply:
Please use godocs locally to ensure these are formatted nicely